Library/Tracking/Koala.swift

```diff
@@ -22,38 +22,57 @@ public final class Koala {
   private let screen: UIScreenType
 
   private enum DataLakeWhiteListedEvent: String, CaseIterable {
-    case explorePageViewed = "Explore Page Viewed"
-    case exploreSortClicked = "Explore Sort Clicked"
     case activityFeedViewed = "Activity Feed Viewed"
-    case editorialCardClicked = "Editorial Card Clicked"
+    case addNewCardButtonClicked = "Add New Card Button Clicked"
+    case checkoutPaymentPageViewed = "Checkout Payment Page Viewed"
     case collectionViewed = "Collection Viewed"
+    case editorialCardClicked = "Editorial Card Clicked"
+    case explorePageViewed = "Explore Page Viewed"
+    case exploreSortClicked = "Explore Sort Clicked"
+    case fbLoginOrSignupButtonClicked = "Facebook Log In or Signup Button Clicked"
     case filterClicked = "Filter Clicked"
-    case tabBarClicked = "Tab Bar Clicked"
-    case searchPageViewed = "Search Page Viewed"
-    case searchResultsLoaded = "Search Results Loaded"
-    case projectSwiped = "Project Swiped"
-    case projectPageViewed = "Project Page Viewed"
+    case forgotPasswordViewed = "Forgot Password Viewed"
+    case loginButtonClicked = "Log In Button Clicked"
     case loginOrSignupButtonClicked = "Log In or Signup Button Clicked"
     case loginOrSignupPageViewed = "Log In or Signup Page Viewed"
-    case fbLoginOrSignupButtonClicked = "Facebook Log In or Signup Button Clicked"
-    case loginButtonClicked = "Log In Button Clicked"
-    case signupButtonClicked = "Signup Button Clicked"
     case loginSubmitButtonClicked = "Log In Submit Button Clicked"
-    case signupSubmitButtonClicked = "Signup Submit Button Clicked"
+    case pledgeSubmitButtonClicked = "Pledge Submit Button Clicked"
     case projectPagePledgeButtonClicked = "Project Page Pledge Button Clicked"
+    case projectPageViewed = "Project Page Viewed"
+    case projectSwiped = "Project Swiped"
+    case searchPageViewed = "Search Page Viewed"
+    case searchResultsLoaded = "Search Results Loaded"
     case selectRewardButtonClicked = "Select Reward Button Clicked"
-    case pledgeSubmitButtonClicked = "Pledge Submit Button Clicked"
-    case checkoutPaymentPageViewed = "Checkout Payment Page Viewed"
+    case signupButtonClicked = "Signup Button Clicked"
+    case signupSubmitButtonClicked = "Signup Submit Button Clicked"
+    case tabBarClicked = "Tab Bar Clicked"
     case thanksPageViewed = "Thanks Page Viewed"
-    case forgotPasswordViewed = "Forgot Password Viewed"
     case twoFactorConfirmationViewed = "Two-Factor Confirmation Viewed"
-    case addNewCardButtonClicked = "Add New Card Button Clicked"
 
     static func allWhiteListedEvents() -> [String] {
       return DataLakeWhiteListedEvent.allCases.map { $0.rawValue }
     }
   }
 
+  /// Determines the screen from which the event is sent.
+  public enum LocationContext: String {
+    case activities = "activity_feed_screen" // ActivitiesViewController <-
```

Comment:
What are the `<-`'s for?

Reply:
Oh, I was using this to keep track of locations that had been tested and didn't clean it up 🤦🏻‍♂️.